velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -64,57 +64,73 @@ struct ScopedOptions {
 
 // Generate random values for the different supported types.
 template <typename T>
-T rand(FuzzerGenerator&) {
+T rand(FuzzerGenerator& rng, DataBoundary dataBoundary = {true, true}) {
   VELOX_NYI();
 }
 
 template <>
-int8_t rand(FuzzerGenerator& rng) {
+int8_t rand(FuzzerGenerator& rng, DataBoundary dataBoundary) {
```

Comment:
nit: int8_t rand(FuzzerGenerator& rng, DataSpec /*dataSpec*/) since dataSpec is unused.